packages/app-distribution/lib/modular/index.js

```diff
@@ -0,0 +1,44 @@
+import { firebase } from '..';
+
+/**
+ * @param {import('@react-native-firebase/app').ReactNativeFirebase.FirebaseApp | undefined} app
+ * @returns {import('.').AppDistribution}
```

Comment:
The params and returns in this file do not seem to make sense to me as they are programmatic? I am expecting something more like this ?

```suggestion
 * @param {FirebaseApp | undefined} app
 * @returns {AppDistribution}
```